test/test_abstract_operations.jl

```diff
@@ -115,6 +116,20 @@ for arch in archs
             end
         end
 
+        @testset "KernelFunctionOperations [$(typeof(arch))]" begin
```

Comment:
Out of curiosity, where does the arch appear in each test?

Reply:
In this file there's an outer loop over `archs`: 

https://github.com/CliMA/Oceananigans.jl/blob/7094b287dad338ea2cd037264fba896499650b8b/test/test_abstract_operations.jl#L78

but not sure about other files.